diff -urN gmic-1.5.2.2/zart/zart.pro gmic-1.5.2.2-patched/zart/zart.pro
--- gmic-1.5.2.2/zart/zart.pro	2012-11-12 01:18:42.000000000 +1100
+++ gmic-1.5.2.2-patched/zart/zart.pro	2012-11-15 17:54:50.398517443 +1100
@@ -36,8 +36,8 @@
 FORMS = ui/MainWindow.ui ui/DialogAbout.ui ui/DialogLicence.ui
 
 !macx {
-	LIBS += -lX11 ../src/libgmic.a -lcxcore -lcv -lml -lhighgui -lfftw3
-#	LIBS += -lX11 ../src/libgmic.a -lopencv_core -lopencv_highgui -lopencv_imgproc -lopencv_objdetect -lfftw3
+#	LIBS += -lX11 ../src/libgmic.a -lcxcore -lcv -lml -lhighgui -lfftw3
+	LIBS += -lX11 ../src/libgmic.a -lopencv_core -lopencv_highgui -lopencv_imgproc -lopencv_objdetect -lfftw3
 } else {
 	LIBS += -lX11 ../src/libgmic.a `pkg-config opencv --libs` -lfftw3
 }
