--- configure.link	2010-11-17 12:20:06.000000000 +0100
+++ configure	2011-03-24 06:41:58.000000000 +0100
@@ -4976,7 +4976,7 @@
   $as_echo_n "(cached) " >&6
 else
   ac_check_lib_save_LIBS=$LIBS
-LIBS="-lGraphicsMagick++  $LIBS"
+LIBS="-lGraphicsMagick++ -lGraphicsMagick $LIBS"
 cat confdefs.h - <<_ACEOF >conftest.$ac_ext
 /* end confdefs.h.  */
 
@@ -5011,7 +5011,7 @@
 #define HAVE_LIBGRAPHICSMAGICK__ 1
 _ACEOF
 
-  LIBS="-lGraphicsMagick++ $LIBS"
+  LIBS="-lGraphicsMagick++ -lGraphicsMagick $LIBS"
 
 else
 
@@ -5074,13 +5074,13 @@
 
 fi
 
-{ $as_echo "$as_me:${as_lineno-$LINENO}: checking for main in -lcv" >&5
-$as_echo_n "checking for main in -lcv... " >&6; }
+{ $as_echo "$as_me:${as_lineno-$LINENO}: checking for main in -lopencv_core" >&5
+$as_echo_n "checking for main in -lopencv_core... " >&6; }
 if test "${ac_cv_lib_cv_main+set}" = set; then :
   $as_echo_n "(cached) " >&6
 else
   ac_check_lib_save_LIBS=$LIBS
-LIBS="-lcv  $LIBS"
+LIBS="-lopencv_core  $LIBS"
 cat confdefs.h - <<_ACEOF >conftest.$ac_ext
 /* end confdefs.h.  */
 
@@ -5109,7 +5109,7 @@
 #define HAVE_LIBCV 1
 _ACEOF
 
-  LIBS="-lcv $LIBS"
+  LIBS="-lopencv_core $LIBS"
 
 else
 
@@ -5120,13 +5120,13 @@
 
 fi
 
-{ $as_echo "$as_me:${as_lineno-$LINENO}: checking for main in -lhighgui" >&5
-$as_echo_n "checking for main in -lhighgui... " >&6; }
+{ $as_echo "$as_me:${as_lineno-$LINENO}: checking for main in -lopencv_highgui" >&5
+$as_echo_n "checking for main in -lopencv_highgui... " >&6; }
 if test "${ac_cv_lib_highgui_main+set}" = set; then :
   $as_echo_n "(cached) " >&6
 else
   ac_check_lib_save_LIBS=$LIBS
-LIBS="-lhighgui  $LIBS"
+LIBS="-lopencv_highgui  $LIBS"
 cat confdefs.h - <<_ACEOF >conftest.$ac_ext
 /* end confdefs.h.  */
 
@@ -5155,7 +5155,7 @@
 #define HAVE_LIBHIGHGUI 1
 _ACEOF
 
-  LIBS="-lhighgui $LIBS"
+  LIBS="-lopencv_highgui $LIBS"
 
 else
 
--- src/Makefile.link	2010-11-17 12:20:06.000000000 +0100
+++ src/Makefile	2011-03-24 07:46:34.000000000 +0100
@@ -129,14 +129,14 @@
 
 # Flags to enable native support of webcams, using the OpenCV library.
 OPENCV_CFLAGS = -Dcimg_use_opencv -I$(USR)/include/opencv
-OPENCV_LDFLAGS = -lcv -lhighgui
+OPENCV_LDFLAGS = -lopencv_core -lopencv_highgui
 
 # Flags to enable native support of most classical image file formats, using the GraphicsMagick++ library.
 MAGICK_CFLAGS = -Dcimg_use_magick -I$(USR)/include/GraphicsMagick
 ifeq ($(OS),Darwin)
 MAGICK_LDFLAGS = -L$(USR)/lib -lGraphicsMagick++ -lGraphicsMagick -llcms -ltiff -lfreetype -ljpeg -lpng -lbz2 -lxml2 -lz -lm -lpthread -lltdl
 else
-MAGICK_LDFLAGS = -lGraphicsMagick++
+MAGICK_LDFLAGS = -lGraphicsMagick++ -lGraphicsMagick
 endif
 
 # Flags to enable native support of EXR file format, using the OpenEXR library/
@@ -274,7 +274,7 @@
 gmic_gimp.o: gmic.cpp gmic_def.h
 	$(CC) -o gmic_gimp.o -c gmic.cpp -Dgmic_gimp -Dgmic_minimal -Dgmic_float $(CFLAGS)
 gmic_gimp : gmic_gimp.o gmic_gimp.cpp
-	$(CC) -o gmic_gimp gmic_gimp.cpp gmic_gimp.o `gimptool-2.0$(EXE) --cflags` $(CFLAGS) `gimptool-2.0$(EXE) --libs` $(LDFLAGS)
+	$(CC) -o gmic_gimp gmic_gimp.cpp gmic_gimp.o `gimptool-2.0$(EXE) --cflags` $(CFLAGS) $(LDFLAGS) `gimptool-2.0$(EXE) --libs`
 	strip gmic_gimp$(EXE)
 
 gmic_minimal: gmic.cpp
