--- gmic-1.5.2.1/src/Makefile.orig	2012-11-03 16:48:20.157000237 +0100
+++ gmic-1.5.2.2/src/Makefile	2012-11-03 16:49:40.028000237 +0100
@@ -268,7 +268,7 @@
 	$(MAKE) linux
 	$(MAKE) gimp
 	$(MAKE) lib
-	$(MAKE) zart
+#	$(MAKE) zart
 else
 ifeq ($(OS),Darwin)
 	@echo "**"
@@ -396,11 +396,11 @@
 	mkdir -p $(DESTDIR)$(USR)/share/zart
 	cp -f ../zart/haar*.xml $(DESTDIR)$(USR)/share/zart
 	cp -f ../zart/zart $(DESTDIR)$(USR)/bin/zart
-	mkdir -p $(DESTDIR)$(USR)/lib
-	cp -f libgmic.so $(DESTDIR)$(USR)/lib/libgmic.so.1.5.2.2
-	ln -s libgmic.so.1.5.2.2 $(DESTDIR)$(USR)/lib/libgmic.so.1.5.2
-	ln -s libgmic.so.1.5.2 $(DESTDIR)$(USR)/lib/libgmic.so.1.5
-	ln -s libgmic.so.1.5 $(DESTDIR)$(USR)/lib/libgmic.so.1
+	mkdir -p $(DESTDIR)$(USR)/@LIB@
+	cp -f libgmic.so $(DESTDIR)$(USR)/@LIB@/libgmic.so.1.5.2.2
+	ln -s libgmic.so.1.5.2.2 $(DESTDIR)$(USR)/@LIB@/libgmic.so.1.5.2
+	ln -s libgmic.so.1.5.2 $(DESTDIR)$(USR)/@LIB@/libgmic.so.1.5
+	ln -s libgmic.so.1.5 $(DESTDIR)$(USR)/@LIB@/libgmic.so.1
 endif
 	mkdir -p $(DESTDIR)$(USR)/share/man/
 	mkdir -p $(DESTDIR)$(USR)/share/man/man1/
@@ -413,10 +413,10 @@
 uninstall:
 	rm -f $(DESTDIR)$(USR)/bin/gmic
 	rm -f $(DESTDIR)$(USR)/include/gmic.h
-	rm -f $(DESTSIR)$(USR)/lib/libgmic.so.1.5.2.2
-	rm -f $(DESTSIR)$(USR)/lib/libgmic.so.1.5.2
-	rm -f $(DESTSIR)$(USR)/lib/libgmic.so.1.5
-	rm -f $(DESTSIR)$(USR)/lib/libgmic.so.1
+	rm -f $(DESTSIR)$(USR)/@LIB@/libgmic.so.1.5.2.2
+	rm -f $(DESTSIR)$(USR)/@LIB@/libgmic.so.1.5.2
+	rm -f $(DESTSIR)$(USR)/@LIB@/libgmic.so.1.5
+	rm -f $(DESTSIR)$(USR)/@LIB@/libgmic.so.1
 	rm -rf $(DESTDIR)$(USR)/share/doc/gmic/
 	rm -f $(DESTDIR)$(USR)/share/man/man1/gmic.1.gz
 	rm -f $(DESTDIR)$(USR)/share/man/fr/man1/gmic.1.gz
